mmv1/third_party/terraform/services/storage/resource_storage_bucket_test.go.tmpl

```diff
@@ -18,6 +18,103 @@ import (
 	"google.golang.org/api/storage/v1"
 )
 
+func TestAccStorageBucket_basic_hns(t *testing.T) {
+	t.Parallel()
+
+	// test case with hns enabled
+	bucketName := acctest.TestBucketName(t)
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccStorageBucketDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccStorageBucket_basic_hns(bucketName, true),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"google_storage_bucket.bucket", "hierarchical_namespace.0.enabled", "true"),
+				),
+			},
+			{
+				ResourceName:            "google_storage_bucket.bucket",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"force_destroy"},
+			},
+		},
+	})
+
+	// test case with hns disabled
```

Comment:
Can you structure this as a separate TestAcc entry? I think that's the cause of the VCR replacing error, we don't expect multiple cases inside the same `t` scope.

Reply:
moved to a separate testcase

Reply:
What is need of separate test for `false` value when you already wrote one test with the same step? I think because of ForceNew behavior, The above update test will create new bucket resource so this test will be redundant.

Reply:
agrees removed the false test case